.github/workflows/build.yml

```
@@ -28,6 +28,30 @@ jobs:
       - name: "Run pnpm build"
         run: pnpm build
 
+      - name: Setup hosts file
+        run: |
+          sudo bash -c "echo '127.0.0.1 holder' >> /etc/hosts"
+          sudo bash -c "echo '127.0.0.1 issuer' >> /etc/hosts"
+          sudo bash -c "echo '127.0.0.1 faber' >> /etc/hosts"
```

Looks like hardcoded values.

An alternative would be to leverage `jq`:

```suggestion
for service in $(jq -r '.services | keys[]' docker/docker-compose.yml); do
    sudo bash -c "echo '127.0.0.1 $service' >> /etc/hosts"
done
```

---

jq does not work with yaml. use grep